src/main/java/org/generationcp/middleware/dao/ims/ExperimentTransactionDAO.java

```diff
@@ -97,7 +97,7 @@ public Long countPlantingTransactionsByStatus(final List<Integer> ndExperimentId
 			return criteria.list();
 		} catch (final HibernateException e) {
 			final String message =
-				"Error at getTransactionsByInstanceIds instanceIds = " + instanceIds + ", transactionType + "
+				"Error at getTransactionsByInstanceId instanceIds = " + instanceIds + ", transactionType + "
```

Comment:
The correct method name is: `getTransactionsByInstanceIds `

Reply:
Done